x-pack/plugins/case/server/client/index.ts

```diff
@@ -8,36 +8,51 @@ import { CaseClientFactoryArguments, CaseClient } from './types';
 import { create } from './cases/create';
 import { update } from './cases/update';
 import { addComment } from './comments/add';
+import { getFields } from './configure/get_fields';
+import { getMappings } from './configure/get_mappings';
 
 export { CaseClient } from './types';
 
 export const createCaseClient = ({
-  savedObjectsClient,
-  request,
   caseConfigureService,
   caseService,
+  connectorMappingsService,
+  request,
+  savedObjectsClient,
   userActionService,
 }: CaseClientFactoryArguments): CaseClient => {
   return {
     create: create({
-      savedObjectsClient,
-      request,
       caseConfigureService,
       caseService,
+      connectorMappingsService,
+      request,
+      savedObjectsClient,
       userActionService,
     }),
     update: update({
-      savedObjectsClient,
-      request,
       caseConfigureService,
       caseService,
+      connectorMappingsService,
+      request,
+      savedObjectsClient,
       userActionService,
     }),
     addComment: addComment({
-      savedObjectsClient,
+      caseConfigureService,
+      caseService,
+      connectorMappingsService,
       request,
+      savedObjectsClient,
+      userActionService,
+    }),
+    getFields: getFields(),
+    getMappings: getMappings({
```

Comment:
Do we want to expose those functionalities to external kibana's plugins? I fine as it is for the moment but maybe is something we need to discuss. @XavierM what do you think?

Reply:
Also it would be nice to have export them as

```suggestion
    connectors: {
        getFields: getFields(),
        getMappings: getMappings({
            caseConfigureService,
            caseService,
            connectorMappingsService,
            request,
            savedObjectsClient,
            userActionService,
        })
    }
```